docs/draft/etl-pipeline-debug-guide-v1.rst

```diff
@@ -0,0 +1,126 @@
+ETL Pipeline Debug Guide V1
+==============================================================================
+
+
+Check Status of the Pipeline
+------------------------------------------------------------------------------
+
+
+Do we have data in Redshift?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This sql scripts gives you counts of rows we copied every days. You can visually check if there is missing data. If you observed significant drop in any date, it means that **ETL Pipeline is having problems.**
```

Comment:
This must be run manually? If so, please indicate that.

Reply:
No, we can create a lambda function to automate that. I created a user story on Jan 31, https://github.com/18F/identity-analytics-etl/issues/181, but we all believe that it is a low priority.

Reply:
As is now, then you mean manually - so please document that.

Reply:
updated docs